Packs/Core/Playbooks/playbook-XCloud_Cryptomining.yml

```diff
@@ -192,9 +192,9 @@ tasks:
           XSOAR has detected activity related to Cryptojacking.
           After careful analysis, XSOAR found the activity malicious.
 
-          Please enter XSOAR incident ID ${incident.id} for further investigation and response.
+          Please enter XSOAR incident ID ${alert.id} for further investigation and response.
```

Comment:
need to change this whole sentence **XSOAR incident ID**

Reply:
fixed